### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by FlorianEsn**

### CODE REVIEW

**Review by Atila18 — APPROVED** [review]

**Review by PhoolRiteHard — APPROVED** [review]

**Review by L-L-42 — APPROVED** [review]

File: frontend/src/components/Ecran.css
Comment (Xgt72) [inline frontend/src/components/Ecran.css:1]: You must change all the name of the classes which are not in english, "Bureau" to "desktop", and the names must be in camelCase or kebab-case.

File: frontend/src/components/Leaderboard.css
Comment (Xgt72) [inline frontend/src/components/Leaderboard.css:1]: you must change the name of the class "tableau" to an english word.

File: frontend/src/components/Timer.jsx
Comment (Xgt72) [inline frontend/src/components/Timer.jsx:1]: The name of the component must be the same as the filename.

**Review by Xgt72 — CHANGES_REQUESTED** [review]
You must change the component name Ecran to Screen and also for Ecran.css.
In Ecran.css, you must use english name for the classes.
See all the comments in different files.
At your disposal to discuss.

File: bash.exe.stackdump
Comment (Xgt72) [inline bash.exe.stackdump:1]: Could you remove this file.

File: frontend/index.html
Comment (Xgt72) [inline frontend/index.html:1]: Please, remove the line with script window.addEventListener("load"....

File: frontend/src/components/Suggestion.jsx
Comment (Xgt72) [inline frontend/src/components/Suggestion.jsx:1]: The filename must be the same as the name of the component.

File: frontend/src/components/Star.jsx
Comment (Xgt72) [inline frontend/src/components/Star.jsx:1]: The filename must be the same as the name of the component.

File: frontend/src/components/Contact.jsx
Comment (Xgt72) [inline frontend/src/components/Contact.jsx:1]: The filename must be the same as the name of the component or the opposite.

**Review by Xgt72 — CHANGES_REQUESTED** [review]
You need some change.

**Review by Xgt72 — APPROVED** [review]

**Pull request merged**